main.sh

```diff
@@ -16,4 +16,4 @@
 #
 # Copyright 2023 The Kepler Contributors
 #
-./kind/common.sh
\ No newline at end of file
+./kind/common.sh $1
```

Comment:
since the $1 is mandatory here, do we need to change kepler/hack/cluster-xxx.sh accordingly?

Reply:
https://github.com/sustainable-computing-io/kepler/blob/main/Makefile#L264 yep, this line should also consume from a tag.

Reply:
We can also add a default case of `_kind_up` in `main.sh` so that we don't need to make changes everywhere `main.sh` is called. wdyt?

Reply:
My bad [this](https://github.com/sustainable-computing-io/local-dev-cluster/pull/10#discussion_r1209489913) change will always call `_kind_up`. I think there is no need to make changes where `main.sh` is called.